com.microsoft.mrtk.input/Experimental/SpatialMouse/Interactor/SpatialMouseInteractor.cs

```diff
@@ -0,0 +1,212 @@
+    /// to have a lot of value even if the details haven�t fully been fleshed out. 
+    /// For these types of features, we want the community to see them and get 
+    /// value out of them early enough so to provide feedback. 
+    /// </remarks>
+    [AddComponentMenu("Scripts/Microsoft/MRTK/Input/MRTK Spatial Mouse Interactor")]
+    public class SpatialMouseInteractor : XRRayInteractor, IRayInteractor
+    {
+        [SerializeField, Experimental]
+        [Tooltip("The Input System action used for mouse movement. Must be a Vector2 Control.")]
+        private InputActionProperty mouseMoveAction;
+
+        [SerializeField]
+        [Tooltip("The Input System action used for mouse scrolling. Must be a Vector2 Control.")]
+        private InputActionProperty mouseScrollAction;
+
+        /// <summary>
+        /// The scale factor to apply to the mouse deltas.
+        /// </summary>
+        [SerializeField]
+        [Tooltip("The scale factor to apply to the mouse deltas.")]
+        public float mouseSensitivity = .05f;
+
+        /// <summary>
+        /// The scale factor to apply to the mouse wheel.
+        /// </summary>
+        [SerializeField]
+        [Tooltip("The scale factor to apply to the mouse wheel.")]
+        public float mouseWheelSensitivity = .002f;
+
+        /// <summary>
+        /// The time (in seconds) of no mouse activity before hiding the mouse cursor.
+        /// </summary>
+        [SerializeField]
+        [Tooltip("The time (in seconds) of no mouse activity before hiding the mouse cursor.")]
+        public float mouseHideThreshold = 20.0f;
+
+        /// <summary>
+        /// he time (in seconds) of no mouse activity before reseting the mouse cursor to the center of the FoV.
+        /// </summary>
+        [SerializeField]
+        [Tooltip("The time (in seconds) of no mouse activity before reseting the mouse cursor to the center of the FoV.")]
+        public float mouseResetThreshold = 0.2f;
+
+        private CursorLockMode restoreLockState;
+
+        private float timeSinceLastMouseEvent = Mathf.Infinity;
+
+        /// <summary>
+        /// Returns true if mouse is actively in use. Should be used to show/hide mouse-specific feedback (e.g. cursor).
+        /// </summary>
+        public bool IsInUse => hasSelection || (timeSinceLastMouseEvent < mouseHideThreshold);
+
+        protected override void OnEnable()
+        {
+            base.OnEnable();
+
+            if (mouseMoveAction != null)
+            {
+                mouseMoveAction.action.performed += OnMouseMove;
+                mouseMoveAction.EnableDirectAction();
+            }
+            if (mouseScrollAction != null)
+            {
+                mouseScrollAction.action.performed += OnMouseScroll;
+                mouseScrollAction.EnableDirectAction();
+            }
+
+            restoreLockState = Cursor.lockState;
+            Cursor.lockState =  CursorLockMode.Locked;
+        }
+
```

Comment:
add `/// <inheritdoc />`  so API reference documents pull in base class summaries